test/dummy/db/schema.rb

```diff
@@ -56,8 +55,9 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.text "arguments"
-    t.index ["task_name", "created_at"], name: "index_maintenance_tasks_runs_on_task_name_and_created_at"
     t.integer "lock_version", default: 0, null: false
+    t.index ["task_name", "created_at"], name: "index_maintenance_tasks_runs_on_task_name_and_created_at"
```

Comment:
Oh great yeah sorry I saw the pagination query and missed the fact that `TaskData#runs` was setting the order. So we use it but with this PR it becomes no longer used. We should have a similar ordered one since we now order by status.